Makefile

```diff
@@ -1,8 +1,39 @@
 PROJ_VERSION = 6.3.0
-
-install_proj:
+install_proj: ## Install PROJ and clang (requirements to use proj crate)
 	sudo apt update
-	sudo apt install -y wget build-essential pkg-config sqlite3 libsqlite3-dev clang
-	wget https://github.com/OSGeo/proj.4/releases/download/$(PROJ_VERSION)/proj-$(PROJ_VERSION).tar.gz && tar -xzvf proj-$(PROJ_VERSION).tar.gz
-	cd proj-$(PROJ_VERSION) && ./configure --prefix=/usr && make && sudo make install
+	sudo apt install -y clang
+
+	# Needed only for proj install
+	sudo apt install -y wget build-essential pkg-config sqlite3 libsqlite3-dev
+
+	# remove PROJ system version from packages
+	sudo apt remove libproj-dev
+
+	wget https://github.com/OSGeo/proj.4/releases/download/$(PROJ_VERSION)/proj-$(PROJ_VERSION).tar.gz
+	tar -xzvf proj-$(PROJ_VERSION).tar.gz
+	pushd proj-$(PROJ_VERSION)
+	./configure --prefix=/usr && make
+	sudo make install
+	popd
 	rm -rf proj-$(PROJ_VERSION) proj-$(PROJ_VERSION).tar.gz
+
+fmt: format ## Check formatting of the code (alias for 'format')
+format: ## Check formatting of the code
+	cargo fmt --all -- --check
+
+clippy: lint ## Check quality of the code (alias for 'lint')
+lint: ## Check quality of the code
+	cargo clippy --workspace --all-features --all-targets -- --warn clippy::cargo --allow clippy::multiple_crate_versions --deny warnings
+
+test: ## Launch all tests
+	# Run all the tests of `transit_model` in the entire repository,
+	# activating all features (including `xmllint`), then without features
+	# to make sure that both work
+	cargo test --workspace --all-features
+	cargo test --workspace
```

Comment:
```suggestion
	cargo test --workspace --all-features --all-targets
	cargo test --workspace --all-targets
```

Reply:
As discussed, this removes doctests.
See https://github.com/rust-lang/cargo/issues/6669 and https://doc.rust-lang.org/cargo/commands/cargo-test.html

So adding this and the corresponding lines to just trigger doctests.